doc/source/whatsnew/v0.24.0.txt

```diff
@@ -63,8 +63,7 @@ Removal of prior version deprecations/changes
 Performance Improvements
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
--
--
+- Improved performance of :func:`Series.describe` in case of numeric dtpyes
```

Comment:
can you add the issue number (this pr number as we don't have an issue)

Reply:
OK but I'm unsure about what format is expected. Do you think a link to an external URL (such as [here](https://github.com/pandas-dev/pandas/blob/67e6e6fcd19d1d89cb60abc3a78372bc85fd8e29/doc/source/whatsnew/v0.23.0.txt#L534)) would be appropriate? E.g., `` `pull request #21274 <https://github.com/pandas-dev/pandas/pull/21274/>`_ ``. Or something else?

Reply:
same format as all the others, just use ``:issue:`number` ``

Reply:
I see now that the URL of the issue is translated to the URL of the PR. That's great.